state_machine/modules/json_conf/include/klepsydra/state_machine/kpsr_state_machine.h

```diff
@@ -16,13 +16,13 @@
 
 #pragma once
 
+#include <klepsydra/state_machine/json_config_loader.h>
```

Comment:
json_config_loader can be moved to the cpp too, right?

Reply:
Yes. Copied to `state_machine/tests/state_machine_tests/src/state_machine_test.cpp` too.